lib/routes/release-line.js

```diff
@@ -0,0 +1,19 @@
+const a = require('express-async-handler');
```

Comment:
a?

Reply:
@msluther @fritzmonkey short for async basically. i just didnt like `asyncHandler` since its so long. Ideas?